### PR TITLE
Auto tweaks

### PR DESCRIPTION
**Pull request by OliverW10**

### CODE REVIEW

File: autonomous/autonomous.py [hunk 1]
```diff
@@ -148,8 +148,10 @@ def move(self, tm: float) -> None:
             abs(translation_error) < self.ALLOWED_TRANS_ERROR
             and abs(rotation_error) < self.ALLOWED_ROT_ERROR
         )
+        is_stopped = abs(self.chassis.translation_velocity.norm()) < 0.2
```
Comment (auscompgeek) [inline autonomous/autonomous.py:151]: Isn't the norm of a (real) vector by definition a non-negative real number?
```suggestion
        is_stopped = self.chassis.translation_velocity.norm() < 0.2
```

**Review by auscompgeek — COMMENTED** [review]
lgtm without joystick changes

File: robot.py [hunk 2]
```diff
@@ -111,19 +114,19 @@ def disabledPeriodic(self) -> None:
     def teleopPeriodic(self) -> None:
         # handle chassis inputs
         throttle = scale_value(self.joystick.getThrottle(), 1, -1, 0.1, 1)
-        spin_rate = 1.5
+        spin_rate = 2.0
         joystick_x = (
-            -rescale_js(self.joystick.getY(), deadzone=0.1, exponential=1.5)
+            rescale_js(self.joystick.getX(), deadzone=0.1, exponential=1.5)
```
[the remark this answers is not in the document]
Comment (OliverW10) [reply robot.py]: i think it was driving correctly with this, ill merge it without it and we can sort it out properly later

[the remark this answers is not in the document]
Comment (auscompgeek) [reply robot.py]: See #73

[the remark this answers is not in the document]
Comment (james-ward) [reply robot.py]: Yeah, I agree - the x and y axes of the joystick are not the same as the robot. Any code that doesn't have them flipped is a red flag.

**Pull request merged**